src/content/aur-discord-yukleme.md

```diff
@@ -21,7 +21,7 @@ Resimde de gördüğünüz gibi aramamıza rağmen ekranımıza hiçbir şey gel
 
 <smart-image src="https://the-person-under-this-message.is-inside.me/ohjHT1bc.png"></smart-image>
 
-Resimde de gördüğünüz gibi AUR depolarında sadece Discord değil, bir çok farklı paket ve sürümler mevcut. Eğer discord paketini listede görmüyorsanız biraz aşağı kaydırabilirsiniz. Aynı zamanda eğer Discord Canary'yi kullanmak istiyorsanız, aşağıdaki paketi yükleyebilirsiniz.
+Resimde de gördüğünüz gibi AUR depolarında sadece Discord değil, birçok farklı paket ve sürümler mevcut. Eğer discord paketini listede görmüyorsanız biraz aşağı kaydırabilirsiniz. Aynı zamanda eğer Discord Canary'i kullanmak istiyorsanız, aşağıdaki paketi yükleyebilirsiniz.
```

Comment:
`discord`, like, with the ` (backtick) 😅

```
`discord`
```

Reply:
I thought you put reverse quotes to indicate it. Let me fix it.

Reply:
Yeah, sorry 😅